### PR TITLE
(DOCSP-45775) Release notes for 1.33

### PR DESCRIPTION
**Pull request by JuliaMongo**
- [DOCSP-45775](https://jira.mongodb.org/browse/DOCSP-45775)
- [Release Notes for 1.33](https://deploy-preview-782--docs-atlas-cli.netlify.app/atlas-cli-changelog/)

Hi Sarah, do I need to review RelNotes with anyone from the dev team? Thank you for your review.

### CODE REVIEW

File: source/atlas-cli-changelog.txt [hunk 1]
```diff
@@ -14,6 +14,22 @@
...
+Released 4 December 2024
+
+- Adds the ``--mdbVersion`` and ``--connectWith`` flags in the :ref:`atlas-setup`
+  command.
+- Deprecates the ``--skipMongosh`` flag in the :ref:`atlas-setup` command.
+  When you use the :ref:`atlas-setup` and :ref:`atlas-deployments-setup`
+  commands in interactive mode, the commands now behave in the same way
+  when you choose ``default`` (Deploy with default settings),
+  ``custom`` (Deploy with custom settings), or ``cancel`` (Cancel setup)
+  options.
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt:31]: I'm confused by this statement... what do we mean by "behave in the same way"? Does it mean that all of those choices do the same thing? Or does it mean that all of those choices can't skip mongosh? I'm assuming it's the latter... can we confirm and make it more clear?

Also, minor nit:
```suggestion
  commands in interactive mode, the commands now behave in the same way
  when you choose ``default`` (deploy with default settings),
  ``custom`` (deploy with custom settings), or ``cancel`` (cancel setup)
  options.
```

Comment (JuliaMongo) [reply source/atlas-cli-changelog.txt:31]: Yes, I too was confused, and thought you'd know. I will check with devs.

Comment (andreaangiolillo) [inline source/atlas-cli-changelog.txt:31]:
```suggestion
  - Updates the :ref:`atlas-deployments-setup` interactive mode to use the same questions and default values used by the :ref:`atlas-setup` command.
```

Comment (andreaangiolillo) [reply source/atlas-cli-changelog.txt:31]: This is the PR https://github.com/mongodb/mongodb-atlas-cli/pull/3410 that adds this change. Here we meant that now the interactive modes of `atlas setup` and `atlas deployment setup` are the same (ask the same questions, use the same default values, behave in the same way).

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Review by sarahsimpers — APPROVED** [review]
Copy changes LGTM! Thank you!

If build errors persist, please raise with DOP- these are expected page errors in our submodule content that have always been there and should not prevent building

**Pull request merged**